docker-compose.yml

```
@@ -1,37 +1,52 @@
 version: '3'
 
 services:
+
   db:
+    container_name: db
     image: postgres:12.0
     ports:
-     - "5432:5432"
+      - 5432:5432
     volumes:
-     - ./database_dumps:/app/database_dumps
+      - ./database_dumps:/app/database_dumps
+
   web:
+    container_name: web
     platform: linux/amd64
     build: .
     volumes:
       - ./:/app
+      - ./bash/bashrc.sh:/root/.bashrc
     depends_on:
-     - db
+      - db
     ports:
-     - 8000:8000 # runserver
-     - 8001:8001 # mkdocs serve
-    # Closely match prod server
-    # command: pipenv run gunicorn config.wsgi:application --bind 0.0.0.0:8000 --timeout 3600 --workers 3 --reload --capture-output
-    # Server for better debugging output
-    command: tail -f /dev/null  # do nothing forever. Use fab sh to run the django server
+      - 8000:8000
+    command: sh bash/run-dev.sh
     restart: on-failure
     init: true
     stdin_open: true
     tty: true
     env_file:
       - .env
+    environment:
+      - DATABASE_HOST=db
+      - DATABASE_NAME=postgres
+      - DATABASE_USER=postgres
+
   cli:
+    container_name: cli
     platform: linux/amd64
     build: platformsh-cli
     volumes:
       - ./media:/app/media
       - ./database_dumps:/app/database_dumps
     env_file:
       - .env
+
+  docs:
+    container_name: docs
+    image: squidfunk/mkdocs-material
+    volumes:
+      - .:/docs
+    ports:
+      - 8001:8000
```

```suggestion
      - 8001:8001
```
Is it supposed to be `8001:8001`? That's how we've got it at the minute, but I'm not too clued up on ports 🙂

---

The default port for MKDocs is `8000` (https://www.mkdocs.org/user-guide/configuration/#dev_addr) and that is what this container image uses (https://github.com/squidfunk/mkdocs-material/blob/master/Dockerfile#L98) but this is the same port that we like to use for ETNA.

What this configuration does is map port `8001` on the host machine (your Mac) to `8000` of the MKDocs container because ETNA and MKDocs can't both use the same port.

If you wanted, you could change this to `12345:8000` and then MKDocs would be available to you on `localhost:12345`. This is just a way to make sure they don't conflict as `8000` is a very popular port for containers to expose.

---

Cool! Thanks for the explanation! 😃